### DISCUSSION TITLE
3d Ansicht

### DISCUSSION THREAD

**Discussion by JanParker1410** [AntMeNet/AntMeClassic#56] [Q&A] [answered]
Hallo.
Ich möchte gerne Programmieren lernen und habe das Programm Antme runter geladen. Auch die Workframes und Visual Studio. Leider fehlt beim Programm die 3d Ansicht. Wie kann ich das ändern oder beheben. Danke LG

**Comment by patteki** [comment, marked as answer]
Hallo Jan,
es freut uns zu hören, dass du mit AntMe! programmieren lernen möchtest.
Um die 3D-Ansicht zu aktivieren, gehe bitte im Menüpunkt Einstellungen unter Plugins dann öffnet sich folgendes Fenster:
<img width="429" alt="image" src="https://github.com/AntMeNet/AntMeClassic/assets/13452239/2ed0cd15-5f30-4383-8a75-196d320c6df4">
dort musst du das Plugin "3D-Ansicht" aktivieren.
Sollte dies bei dir nicht funktionieren oder wenn du weitere Probleme/Fragen hast, melde dich gerne noch einmal :)

Viel Erfolg beim Programmieren lernen und frohes Krabbeln
LG Patrick
